src/test/ui/regions/region-lifetime-bounds-on-fns-where-clause.rs

```diff
@@ -11,23 +11,20 @@ fn b<'a, 'b>(x: &mut &'a isize, y: &mut &'b isize) {
     // Illegal now because there is no `'b:'a` declaration.
     *x = *y;
     //[base]~^ ERROR E0623
-    //[nll]~^^ ERROR lifetime may not live long enough
```

Comment:
I think these errors disappear because they are *borrowck* errors, but we now error and bail in typeck.